src/library/Uno.Material/Styles/Controls/Divider.xaml

```diff
@@ -0,0 +1,45 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:converters="using:Uno.Material.Converters"
+					xmlns:controls="using:Uno.Material.Controls"
+					xmlns:toolkit="using:Uno.UI.Toolkit">
+
+	<ResourceDictionary.MergedDictionaries>
+		<ResourceDictionary Source="../Application/Colors.xaml" />
+	</ResourceDictionary.MergedDictionaries>
+
+	<x:Double x:Key="MaterialDividerHeight">1</x:Double>
```

Comment:
According to material's specs, do we want 1pixel height or we want 1 logical pixel ? We can have a finer line if we use the 1pixel line. I needs a little more code though.

Reply:
To be honest, I'm not sure. The documents is not explicit about that. https://material.io/components/dividers#specs. The only thing it said is that its height is "1". After that, I'm not sure if we're talking pixel or logical pixel

Reply:
I did a bit of googling and examples in axml seem to use 1dp which is a density pixel. (same type as uno is using) so we can leave like you did it's good!